.github/workflows/rebuild-released-images.yaml

```diff
@@ -87,8 +82,22 @@ jobs:
         shell: bash
         run: |
           go mod download
+      - name: Choose Dockerfile
+        id: pick-dockerfile
+        run: |
+          if test -f "fast.Dockerfile"; then
+            echo "dockerfile=fast.Dockerfile" >> $GITHUB_OUTPUT
```

Comment:
the concatenation using `>>` looks dangerous, why can't we use `>` ?

Reply:
Seems to be the standard for GitHub outputs:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-masking-a-generated-output-within-a-single-job

So that you do not remove the previous arguments dump in there by mistake.